pandas/_libs/tslibs/period.pyx

```diff
@@ -1217,6 +1217,32 @@ cdef class _Period(object):
 
     @property
     def day(self):
+        """
+        Return the number of days of particular month of a given Period.
+
+        This attribute returns the total number of days of given month on which
```

Comment:
This is a pretty simple function so depending on how concise the short summary is might be OK to not have an extended summary

Reply:
but I think when I remove extended summary validation_docstring.py gives error ???

Reply:
You are right that the script would fail but I think this would pass as an exception